src/cata_string_consts.h

```diff
@@ -4,6 +4,7 @@
 
 #include "type_id.h"
 #include "player_activity.h"
+#include "itype.h"
```

Comment:
Why is this required in this header? There are no other changes to the header, so why is this suddenly required?

Reply:
It could've been something weird, but for some reason cata_string_consts.h started complaining about not knowing what itype was, this fixed it, no idea why it was required or why it started complaining.
Just wouldnt compile without.

Ill try a rebase now.

Reply:
That is due to circular reference of headers (`cata_string_consts.h` <-> `player_activity.h`). I want to to resolve it in #38141.

Reply:
ah ok, thanks.